src/Components/FileExplorer/Component_FileExplorer.re

```diff
@@ -242,19 +245,28 @@ let update = (~configuration, msg, model) => {
 
     let model = {...model, treeView};
     switch (outmsg) {
-    | Expanded(node) => (
+    | Component_VimTree.Expanded(node) => (
         model,
         Effect(
           Effects.load(node.path, configuration, ~onComplete=newNode =>
             NodeLoaded(newNode)
           ),
         ),
       )
-    | Collapsed(_) => (model, Nothing)
-    | Selected(node) =>
+    | Component_VimTree.Collapsed(_) => (model, Nothing)
+    | Component_VimTree.Clicked(node) =>
       // Set active here to avoid scrolling in BufferEnter
-      (model |> setActive(Some(node.path)), OpenFile(node.path))
-    | Nothing => (model, Nothing)
+      (
+        model |> setActive(Some(node.path)),
+        OpenFile({filePath: node.path, preview: true}),
+      )
```

Comment:
I think another consequence of removing `Selected` is that I believe this will make `Enter` in the file explorer open a preview.

Not sure what the desired approach would be there actually.....If I go into the explorer and hit enter on a file, what should happen? Preview or fully open?

Not sure, though I guess the act of double clicking doesn't really exist with an enter key (plus when you hit enter, focus jumps to the buffer, so right now there is no way of not previewing a file via the vim binds).

Reply:
Good point @CrossR  - I think the `Enter` key should behave similar to NerdTree/Ex, so open a file here.